src/glmtools.jl

```diff
@@ -499,6 +499,19 @@ false
 dispersion_parameter(D) = true
 dispersion_parameter(::Union{Bernoulli, Binomial, Poisson}) = false
 
+"""
+    _safer_int(x::T)
```

Comment:
Safer than what? :-) Why not just `_safe_int`?

Reply:
Safer than `Int` itself. I'm not sure it's *safe* per se, but I can rename. It's an internal method anyway